pkg/gardenlet/bootstrap/util/util.go

```diff
@@ -258,7 +258,7 @@ func ComputeGardenletKubeconfigWithServiceAccountToken(ctx context.Context, gard
 func TokenID(meta metav1.ObjectMeta) string {
```

Comment:
Should we perhaps move this into `pkg/utils/kubernetes/bootstraptoken`? It is a bit weird that `gardenadm` imports a `gardenlet`-specific bootstrap package.